test/Benchmarks/Instrumentation/AspNetCoreInstrumentationNewBenchmarks.cs

```diff
@@ -139,9 +139,14 @@ public void GetRequestForAspNetCoreAppGlobalSetup()
             this.StartWebApplication();
             this.httpClient = new HttpClient();
 
+            var exportedItems = new List<Metric>();
             this.meterProvider = Sdk.CreateMeterProviderBuilder()
                 .ConfigureServices(services => services.AddSingleton<IConfiguration>(configuration))
                 .AddAspNetCoreInstrumentation()
+                .AddInMemoryExporter(exportedItems, metricReaderOptions =>
```

Comment:
Could you also update the Benchmark numbers if you ran it locally?

Reply:
no I did not. Won't have enough cycles to try that out, so we can either merge this PR as is, or wait for someone who is actively working on performance, and have them pick the changes from this PR.